### ISSUE TITLE
Bootstrap Transacations Identity coming from Connection Profile not card

### ISSUE THREAD

**Issue by mbwhite** [hyperledger/composer#2557]
The certificate for the start transactions that will be used should the cli/app not provided any is coming from the old profile store not the card. 

Makes it tricky to deploy unless you duplicate the card information in the old format

**Comment by mbwhite** [comment]
Solution is ini adminconnection.js ln 486
```
        return this._getCurrentIdentity()
            .then((identity) => {

                // Extract the current identity name and certificate.
                identityName = this.securityContext.getUser();
                identityCertificate = identity.certificate;

                    // Now serialize the business network archive.
                return businessNetworkDefinition.toArchive();
```
Pass in the card to this fn and then get the certificate from that. Rather than doing an the `getCurrentIdentity` that does an `exportIdentity`

**Comment by mbwhite** [comment]
Resolved in #2567

**Comment by sstone1** [comment]
Closing as per last comment!
